api/prisma/migrations/16_add_duplicates_view/migration.sql

```diff
@@ -0,0 +1,43 @@
+CREATE VIEW "application_flagged_set_possibilities" AS (
+    SELECT
+        LOWER(a.first_name) || '-' || LOWER(a.last_name) || '-' || a.birth_month || '-' || a.birth_day || '-' || a.birth_year as "key",
+        app.listing_id,
+        app.id as "application_id",
+        'nameAndDOB' as "type"
+    from
+        applicant a,
+        applications app
+    where
+        a.id = app.applicant_id
+)
+UNION
+(
+    SELECT
+        a.email_address as "key",
+        app.listing_id,
+        app.id as "application_id",
+        'email' as "type"
+    from
+        applications app,
```

Comment:
really cool querying syntax! looks to be as performant as traditional joins too!